x-pack/plugins/aiops/public/get_document_stats.ts

```diff
@@ -132,24 +143,30 @@ export const processDocumentCountStats = (
       totalCount,
     };
   }
-  const buckets: { [key: string]: number } = {};
+
   const dataByTimeBucket: Array<{ key: string; doc_count: number }> = get(
     randomSamplerWrapper && body.aggregations !== undefined
       ? randomSamplerWrapper.unwrap(body.aggregations)
       : body.aggregations,
     ['eventRate', 'buckets'],
     []
   );
-  each(dataByTimeBucket, (dataForTime) => {
-    const time = dataForTime.key;
-    buckets[time] = dataForTime.doc_count;
-  });
+
+  const buckets = dataByTimeBucket.reduce<Record<string, number>>((acc, cur) => {
```

Comment:
not needed for this PR, but I've refactored this code to replace lodash `each` with a `reduce`